modules/sovrnBidAdapter.js

```diff
@@ -81,15 +81,12 @@ export const spec = {
         };
       }
 
-      if (bidderRequest && bidderRequest.gdprConsent) {
-        sovrnBidReq.regs = {
-          ext: {
-            gdpr: +bidderRequest.gdprConsent.gdprApplies
-          }};
-        sovrnBidReq.user = {
-          ext: {
-            consent: bidderRequest.gdprConsent.consentString
-          }};
+      if (bidderRequest.gdprConsent) {
+        utils.deepSetValue(sovrnBidReq, 'regs.ext.gdpr', +bidderRequest.gdprConsent.gdprApplies);
```

Comment:
how did you come to use deepSetValue. just curious.

Reply:
i think the prebid peeps mentioned it in a previous PR